wfe-web/src/main/java/ru/runa/common/WebResources.java

```diff
@@ -201,4 +205,7 @@ public static boolean isChatEnabled() {
         return RESOURCES.getBooleanProperty("chat.enabled", true);
     }
 
+    public static int getMaxMessageSize() {
+        return RESOURCES.getIntegerProperty("chat.max.message.size", DEFAULT_CHAT_MAX_MESSAGE_SIZE);
```

Comment:
Лучше указать единицы измерения:
```suggestion
        return RESOURCES.getIntegerProperty("chat.max.message.size.bytes", DEFAULT_CHAT_MAX_MESSAGE_SIZE);
```